site/public/js/activity-dashboard.js

```diff
@@ -0,0 +1,123 @@
+/* exported clear, sortTable */
```

Comment:
This whole file could be converted to mjs very easily since it is brand new. This would allow for tests to be written for it which this should have.

Reply:
When I try to move the file into the mjs file and run the page, the console gives me a 403 Forbidden error, do you know how I should fix it?